packages/hooks/src/useEventListener.ts

```diff
@@ -1,7 +1,7 @@
 import { useEffect, useRef } from 'react';
 
 interface UseEventOptions {
-  options?: object;
+  options?: EventListenerOptions;
```

Comment:
No, but I wish it would. This was just a case where I found the existing definition for this type, so I figured I'd narrow it to what can actually be passed to the event listener.